.github/workflows/cs-fix.yml

```diff
@@ -5,18 +5,20 @@ on:
     types: [opened, synchronize, reopened, ready_for_review]
     paths:
     - '**.php'
+  pull_request_target:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths:
+    - '**.php'
 
 jobs:
   php-cs-fixer:
     runs-on: ubuntu-latest
-    # dont run jobs on forks, because we are not allowed to commit
-    if: github.event.pull_request.draft == false && github.repository == 'redaxo/redaxo'
+    if: github.event.pull_request.draft == false
 
     steps:
     - uses: actions/checkout@v2
       with:
-        ref: ${{ github.head_ref }}
-        token: ${{ secrets.STAABM_TOKEN }}
+        ref: ${{ env.GITHUB_SHA }}
```

Comment:
laut dem beispiel von https://github.com/stefanzweifel/git-auto-commit-action#using-the-action-in-forks-from-public-repositories müsse es `ref: ${{ github.head_ref }}` heißen, oder?

Reply:
Wenn ich head_ref verwende schlägt der Checkout fehl: https://github.com/jelleschutter/redaxo/pull/9

Reply:
Mit SHA bricht er erst beim Commit ab: https://github.com/jelleschutter/redaxo/pull/7/checks?check_run_id=1020919517